release/src/main/python-release/run_release_candidate_python_quickstart.sh

```diff
@@ -139,6 +142,7 @@ function verify_wordcount_dataflow() {
 #   None
 #######################################
 function verify_streaming_wordcount_direct() {
+  cleanup_pubsub
```

Comment:
Cleanup_pubsub at beginning may cause errors if the topic or subscription does not exist. Need to add some if-else clause to avoid the error. https://github.com/apache/beam/blob/master/release/src/main/python-release/python_release_automation_utils.sh#L220

Reply:
suppress standard error when calling `gcloud pubsub topics delete`. PTAL